cmd/snap-bootstrap/bootstrap/bootstrap.go

```diff
@@ -142,6 +143,28 @@ func Run(gadgetRoot, device string, options Options) error {
 	return nil
 }
 
+// bytesIEC is a heleper type that implements the Stringer() interface and
+// formats the size using multiples from IEC units (i.e. kibibytes, mibibytes).
+// Printed values are truncated to 2 decimal points.
+type bytesIEC gadget.Size
+
+func (b bytesIEC) String() string {
```

Comment:
Could we use strutil.SizeToStr() for this? It looks a bit similar.

Reply:
I think the chief difference here is that this function outputs IEC units which are multiples of 1024 (2^10), instead of SizeToStr which outputs units which are multiples of 1000 (10^3).

I do agree with the suggestion to use that instead, because up to this point I think all bits/bytes units that we output as strings are multiples of 1000, so staying consistent would be good.

Reply:
The problem I tried to address is that `gadget.yaml` and therefore ubuntu-image, and the gadget package all use 1024s instead of 1000s. So I would prefer to have the size reported here consistent with the gadget spec. I think I'll move the size printing code to the gadget package to keep the relevant bits (and bytes) close.

Reply:
Thanks! I missed that the gadget is using power of 2 instead of power of 10 (which SizeToStr is using). So that is fine then.